theme/pages/installation.html

```diff
@@ -10,7 +10,7 @@ <h6 class="toc__headline">On this page</h6>
         <li class="toc__entry">
             <a href="#installation-for-mezzio-and-laminas-mvc-application" class="toc__link nav-link">
                 Installation for
-                {% if config.extra.installation.config_provider_class %}mezzio{% endif %}
+                {% if config.extra.installation.config_provider_class %}Mezzio{% endif %}
                 {% if config.extra.installation.config_provider_class and config.extra.installation.module_class %}and{% endif %}
                 {% if config.extra.installation.module_class %}laminas-mvc{% endif %}
                 Application
```

Comment:
As noted in Slack:

- add "A" after "for"
- use "or" instead of "and" to join the application types

As such, this would then read:

- "Installation for A Mezzio Application"
- "Installation for A laminas-mvc Application"
- "Installation for A Mezzio or laminas-mvc Application"

Do the same in lines 31-35 below as well.